src/KeyringController.ts

```diff
@@ -443,13 +443,13 @@ class KeyringController extends EventEmitter {
     },
     opts: Record<string, unknown> = {},
   ): Promise<string> {
-    const address = normalizeToHex(msgParams.from);
+    const address = normalizeToHex(msgParams.from) as Hex;
     const keyring = await this.getKeyringForAccount(address);
     if (!keyring.signPersonalMessage) {
       throw new Error(KeyringControllerError.UnsupportedSignPersonalMessage);
     }
 
-    const normalizedData = normalizeToHex(msgParams.data);
+    const normalizedData = normalizeToHex(msgParams.data) as Hex;
```

Comment:
why not have the normalizeToHex return the proper Hex value you are expecting?

Reply:
It's part of another module I have to update, this is just a temporary measure